src/ovp_pipeline/commands/backfill_entities.py

```
@@ -0,0 +1,209 @@
+    extraction_log.parent.mkdir(parents=True, exist_ok=True)
+
+    processed = 0
+    total_mentions = 0
+    total_candidates = 0
+    errors = 0
+    t0 = time.time()
+
+    for i, fpath in enumerate(md_files):
+        try:
+            extraction = extractor.extract_entities_from_file(fpath)
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)